docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -6,6 +6,66 @@ Use this extension to get the full list of problems found by Codacy for a pull r
 
 !!! info "To use this extension you need a [Codacy account](https://www.codacy.com/signup-codacy)"
 
+## Interface overview
+
+The main view of the extension displays information about the code quality and coverage changes introduced by the pull requests you're working on, as well as the quality status of analyzed branches. This information is grouped into three panels:
+
+-   [Pull request status](#status-tab)
+-   [Open pull requests](#open-pull-requests-tab)
+-   [Analyzed branch](#analyzed-branch-tab)
+
+![Important issues](./images/codacy-vscode-extension-main-view.png)
+
+### Status tab
+
+The **Pull request status** tab displays the following information for the pull request of the currently checked-out branch:
+
+-   The **Quality status** of the pull request, either up to standards or not up to standards, based on the [Quality gates](../repositories-configure/adjusting-quality-gates.md) set for the repository.
+
+-   Any **Quality issues** introduced or fixed by the pull request. These are the same issues you find in the [Quality Issues tabs](../repositories/pull-requests.md#issues-tabs) in the Codacy app and are also visible in VS Code's Problems tab.
+
+    The number next to each file name is the total number of Quality issues that the pull request adds to or removes from that file. The number farther to the right is the number of new Medium or Critical Quality issues. If there are any such issues, the file name is also highlighted in yellow (Medium) or red (Critical).
```

Comment:
The number farther to the right is actually added by VSCode, and it's the amount of problems on that file. Problems that may or may not come from Codacy.

Reply:
Done 👍